crates/autopilot/src/run_loop.rs

```diff
@@ -352,6 +355,7 @@ impl RunLoop {
         self.persistence
             .store_order_events(order_uids, OrderEventLabel::Ready);
 
+        let _timer = Metrics::get().solvers_response_time.start_timer();
```

Comment:
How is this different to the `solve_ok` and `solve_err` metric we already collect?

Reply:
The current metric is about the individual solver's time. Might be redundant, though.